edx/edx-enterprise
# [ENT-428] Remove hard requirement on `configuration_helpers`.

**Description:** This PR removes our hard requirement on `configuration_helpers` by abstracting all uses of it into a single utility function which can be easily modified in the future in case we decide to officially drop the need for `configuration_helpers`.

**JIRA:** [ENT-428](https://openedx.atlassian.net/browse/ENT-428)

**Merge deadline:** N/A

**Testing instructions:**

Just see that the abstraction is occurring properly and that tests are still passing.

To be safe, go to the course enrollment landing page with this PR's code installed, and make sure the platform name and other values we get through `configuration_helpers` are still gotten.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
